src/devices/Card/README.md

```diff
@@ -0,0 +1,25 @@
+# RFID shared elements
+
+Common elements of all RFID and NFC readers like the type of cards 14443 Type A, Type B, Innovision Jewel. 
+
+This contains as well an absctract class that has to be implemented by readers to have a transparent way of managing Mifare cards and other high level cards.
+
+```csharp
+/// <summary>
+/// Abstract class implementing a specific Write and Read function
+/// This class has to be implemented in all RFID/NFC readers
+/// So Mifare cards can be used the same way independant of any reader
+/// </summary>
+public abstract class RfidWriteRead
```

Comment:
I wonder if this should be called something like RfidTransceiver or something along these lines. Current name feels a bit awkward but I'm not super familiar with this to propose better name

Reply:
Yes, you're right. Changed everywhere for ```CardTransceiver``` and function ```Transcieve```.